middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -250,7 +251,8 @@ let close_const0 acc dbg (const : Lambda.structured_constant) =
         | Naked_int32 _ -> KS.naked_int32
         | Naked_int64 _ -> KS.naked_int64
         | Naked_nativeint _ -> KS.naked_nativeint
-        | Naked_vec128 _ -> KS.naked_vec128)
+        | Naked_vec128 _ -> KS.naked_vec128
+        | Null -> KS.any_value (* CR layouts v3: give a more precise kind? *))
```

Comment:
Note: this is correct, if not very precise, and it turns out these kinds aren't used anyway so I will delete this code in a follow-up PR.